packages/vitest-react-native/setup.js

```diff
@@ -75,6 +75,22 @@ const cacheExists = (cachePath) => fs.existsSync(cachePath)
 const readFromCache = (cachePath) => fs.readFileSync(cachePath, 'utf-8')
 const writeToCache = (cachePath, code) => fs.writeFileSync(cachePath, code)
 
+const processBinary = (code, filename) => {
+  const b64 = Buffer.from(code).toString('base64')
+  return `Buffer.from("${b64}")`
+}
+
+addHook(
+  (code, filename) => {
+    return processBinary(code, filename)
+  },
+  {
+    exts: [".png", ".jpg"],
+    ignoreNodeModules: false,
+    matcher: () => true
```

Comment:
I don't think you need a matcher

Reply:
Hmmm... don't know why it failed before but you were right 😮 Removed it now